src/pages/Home/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const Home = () => {
+  return (
+    <div className="h-full">
+      <div
+        className="relative flex items-center justify-center text-center bg-cover bg-center h-65 xl:h-70"
+        style={{
+          backgroundImage: "url(/assets/happy-woman-with-smartphone.jpg)",
+        }}
```

Comment:
you can use [arbitrary](https://tailwindcss.com/docs/background-image#arbitrary-values) values for it with the tailwind. and also you can [configure](https://tailwindcss.com/docs/background-image#customizing-your-theme) it.

Reply:
Let's try both these ways